chat-plugins/info.js

```diff
@@ -165,7 +165,8 @@ exports.commands = {
 	ipsearchall: 'ipsearch',
 	hostsearch: 'ipsearch',
 	ipsearch: function (target, room, user, connection, cmd) {
-		if (!target.trim()) return this.parse('/help ipsearch');
+		if (!target.trim() || !target.includes('.')) return this.parse('/help ipsearch');
```

Comment:
https://nodejs.org/api/net.html#net_net_isip_input

> net.isIP(input)#
Added in: v0.3.0
Tests if input is an IP address. Returns 0 for invalid strings, returns 4 for IP version 4 addresses, and returns 6 for IP version 6 addresses.